index.html

```diff
@@ -13,11 +15,11 @@
       <h2>The Main Content</h2>
       <div class='left'>
       	This is the left side
-      	<img src='https://jesse.sh/img/me.jpg'>
+      	<img src='https://jesse.sh/img/me.jpg' alt="Jssesse image displayed">
       </div>
-      <div class='right'>
+      <p class='right'>
```

Comment:
While this is more semantic, it is not an accurate use of the `<p>`. It stands for paragraph, meaning that it should only contain next. It should not act as a container like `<div>` does.

I would recommend using `<section>` instead here.